portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java

```diff
@@ -547,12 +547,8 @@ protected void renameDuplicateTitle(DLFileEntry dlFileEntry)
 
 				return;
 			}
-			catch (PortalException pe) {
-				if (!(pe instanceof DuplicateFolderNameException) &&
-					 !(pe instanceof DuplicateFileException)) {
-
-					throw pe;
-				}
+			catch (DuplicateFileException | DuplicateFileEntryException |
```

Comment:
@adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside validateFile